modules/tinymce/src/core/main/ts/textpatterns/core/InlinePattern.ts

```diff
@@ -260,7 +260,7 @@ const findPatterns = (editor: Editor, patterns: InlinePattern[], space: boolean)
   }
 
   return Utils.getParentBlock(editor, rng).bind((block) => {
-    const offset = rng.startOffset - (space ? 1 : 0);
+    const offset = rng.startOffset > 0 ? rng.startOffset - (space ? 1 : 0) : rng.startOffset;
```

Comment:
how about
```suggestion
    const offset = Math.max(0, rng.startOffset - (space ? 1 : 0));
```

Reply:
That'd work 👍  I had originally used `Num.clamp` but it felt like overkill because we don't need to check the upper bounds.